_rules/links-with-identical-names-and-context-serve-equivalent-purpose-fd3a94.md

```diff
@@ -0,0 +1,298 @@
+```
+
+#### Passed Example 3
+
+Resources are not the same, since the links resolve to different URLs, but the resources are completely identical, thus serving the same purpose:
+
+```html
+<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/index.html">Contact us</a>
+<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/index-copy.html">Contact us</a>
+```
+
+#### Passed Example 4
+
+Same link text used for links going to pages where the content section is the same, but where the navigation options (bread crumbs and local sub menus) differ due to different placement in navigation hierarchy:
+
+```html
+<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/about/contact.html">Contact us</a>
+<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/careers/contact.html">Contact us</a>
+```
+
+#### Passed Example 5
+
+URLs differ due to trailing slashes, but resolves to the same resource after redirects caused by user agent:
+
+```html
+<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/">Contact us</a>
+<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66">Contact us</a>
+```
+
+#### Passed Example 6
+
+Pages contain different amounts of information and/or differently worded information, but fulfil the same purpose in relation to the link:
+
+```html
+<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/page1.html">Call us</a>
+<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/page2.html">Call us</a>
+```
+
+#### Passed Example 7
+
+Pages have the same advertised key content but use different layouts:
+
+```html
+<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/page1.html">Contact us</a>
+<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/page3.html">Contact us</a>
+```
+
+#### Passed Example 8
+
+Links created via scripting with explicit role of link, but lead to the same resource:
+
+```html
+<span
+	role="link"
+	tabindex="0"
+	onclick="location='/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/index.html'"
+>
+	Link text
+</span>
+
+<span
+	role="link"
+	tabindex="0"
+	onclick="location='/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/index.html'"
+>
+	Link text
+</span>
+```
+
+#### Passed Example 9
+
+A set of two SVG `<a>` elements have the same accessible name and link to the same resource.
+
+```html
+<svg
+	viewBox="0 0 100 100"
+	xmlns="http://www.w3.org/2000/svg"
+	xmlns:xlink="http://www.w3.org/1999/xlink"
+>
+	<a href="http://facebook.com" aria-label="Follow us">
+		<circle cx="50" cy="40" r="35" />
+	</a>
+
+	<a href="http://facebook.com">
+		<text x="50" y="90" text-anchor="middle">
+			Follow us
+		</text>
+	</a>
+</svg>
+```
+
+### Failed
+
+#### Failed Example 1
+
```

Comment:
aren't all the listed failed examples exempt since those links would be confusing/ambiguous to *all* users? e.g. how would a sighted mouse user not be confused by them versus an AT user?

Reply:
That rule is Work in Progress, so anything in it might need to change ;-)
(and locally, it's waiting for #555 to be finished so that we can reuse the definition of context for the links…)